From: Gregory CLEMENT <gregory.clement@free-electrons.com>
Date: Mon, 8 Feb 2016 18:14:10 +0100
Subject: irqchip/armada-370-xp: Do not enable it by default when ARCH_MVEBU is
 selected
Git-commit: 63131b636a0ec969e8b98122c1a928b5a2649d3b
Patch-mainline: v4.6-rc1
References: fate#319899 fate#320700

The irq-armada-370-xp driver can only be built for ARM 32 bits. The mvebu
family had grown with a new ARM64 SoC which will also select the
ARCH_MEVBU configuration. Since "ARM: mvebu: use the ARMADA_370_XP_IRQ
option", the ARM32 mvebu SoC directly select this new option. Selecting
it by default when ARCH_MEVBU is selected is no more needed.

This patch removes this dependency, thanks to this, a kernel for ARM64
mvebu SoC can be built without error due this driver.

Signed-off-by: Gregory CLEMENT <gregory.clement@free-electrons.com>
Link: https://lkml.kernel.org/r/1454951660-13289-3-git-send-email-gregory.clement@free-electrons.com
Signed-off-by: Jason Cooper <jason@lakedaemon.net>
Signed-off-by: Matthias Brugger <mbrugger@suse.com>
---
 drivers/irqchip/Kconfig |    1 -
 1 file changed, 1 deletion(-)

--- a/drivers/irqchip/Kconfig
+++ b/drivers/irqchip/Kconfig
@@ -49,7 +49,6 @@ config ARM_VIC_NR
 
 config ARMADA_370_XP_IRQ
 	bool
-	default y if ARCH_MVEBU
 	select GENERIC_IRQ_CHIP
 
 config ATMEL_AIC_IRQ
